README.adoc

```diff
@@ -14,9 +25,30 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
+== Introduction
+Organice is a desktop Organ Transplant Manager
```

Comment:
Perhaps a better way to word it could be 'desktop organ transplant management application'?

Reply:
Do you think it will be better if I split it into 2 lines? i.e.:
ORGANice works as an organ transplant manager. It can be used on desktop platform.